pgpubsub/listen.py

```diff
@@ -120,8 +126,9 @@ def process(self):
         notification = (
             Notification.objects.select_for_update(
                 skip_locked=True).filter(
+                Q(payload=CastToJSONB(Value(self.notification.payload)))
```

Comment:
I would like get rid of filtering by the payload content here. Basically I would like to implement what is suggested in https://github.com/Opus10/django-pgpubsub/issues/67 that is fetching by the notification `id` which should be passed in the payload instead of the serialized json.

I am not sure it makes sense to do that in this MR though.

Reply:
Agree that this type of filtering seems suboptimal. I'm sure there was a reason I chose not to use the id, but I can't quite remember what that was. One reason may have been if the stored Notification which sent the NOTIFY got deleted somehow and then replaced with another with the same payload (but of course different id). Then the first transient NOTIFY event could still pick up the first stored Notification. That doesn't really seem like a good enough reason though.
But also agree that change should be in a separate PR.